target/a72q/patches/ascendia/customize.sh

```diff
@@ -0,0 +1,37 @@
+SKIPUNZIP=1
+
+# [
+ASCENDIA_IMG="https://downloads.simon1511.de/s/nN65wd7AdtBg95A/download/Ascendia_1.0_Vanilla_OneUI_a72q_boot.img"
+
+REMOVE_FROM_WORK_DIR()
+{
+    local FILE_PATH="$1"
+
+    if [ -e "$FILE_PATH" ]; then
+        local FILE
+        local PARTITION
+        FILE="$(echo -n "$FILE_PATH" | sed "s.$WORK_DIR/..")"
+        PARTITION="$(echo -n "$FILE" | cut -d "/" -f 1)"
+
+        echo "Debloating /$FILE"
+        rm -rf "$FILE_PATH"
+    fi
+}
+
+REPLACE_KERNEL_BINARIES()
+{
+    [ -d "$TMP_DIR" ] && rm -rf "$TMP_DIR"
+    mkdir -p "$TMP_DIR"
+
+    echo "Downloading $(basename "$ASCENDIA_IMG")"
+    curl -L -s -o "$TMP_DIR/boot.img" "$ASCENDIA_IMG"
+
+    echo "Extracting kernel binaries"
+    rm -f "$WORK_DIR/kernel/"*.img
+    cp "$TMP_DIR/boot.img" "$WORK_DIR/kernel/"
```

Comment:
Remove only boot.img from the kernel dir/adjust cp cmd:
https://github.com/BlackMesa123/UN1CA/blob/967b129dfa4823e380da02033d00c553ef4cb325/target/a54x/patches/kernelsu/customize.sh#L15-L18

Reply:
Why? Im directly downloading the boot image and moving it, theres no extra stuff that needs to be moved.

Reply:
done